### ISSUE TITLE
[Dataset]

### ISSUE THREAD

**Issue by GriffinLiang** [hilbysfe/brain-vessel-segmentation#1]
Hi, whether the dataset used in your paper will be publicly available?

**Comment by hilbysfe** [comment]
Hi,Due to data protection laws the imaging data used in this work can not be made available. However, in a shortly upcoming publication we include results and Ground truth Segmentations for an external, public dataset. Here is the link for the accepted publication:https://www.frontiersin.org/articles/10.3389/frai.2020.552258/abstract Best,Adam   Von: Kongming LiangGesendet: Dienstag, 11. August 2020 13:33An: hilbysfe/brain-vessel-segmentationCc: SubscribedBetreff: [hilbysfe/brain-vessel-segmentation] [Dataset] (#1) Hi, whether the dataset used in your paper will be publicly available?—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by GriffinLiang** [comment]
Thanks for your reply. I have read the paper. Do you mean one of the three datasets PEGASUS, 7UP, and 1000Plus?

**Comment by hilbysfe** [comment]
No, the external validation set is part of the MIDAS data collection. Link to data and to labels is available in the paper I linked above. Best,Adam Von: Kongming LiangGesendet: Mittwoch, 16. September 2020 11:32An: hilbysfe/brain-vessel-segmentationCc: Adam Hilbert; CommentBetreff: Re: [hilbysfe/brain-vessel-segmentation] [Dataset] (#1) Thanks for your reply. I have read the paper. Do you mean one of the three datasets PEGASUS, 7UP, and 1000Plus?—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by ivanka1996** [comment]
Hi, very thank you for you shared. whether the model weight trained from your paper will be publicly available?

**Comment by KevinAquino** [comment]
Hi I would like to know this too - will the model be publicly be available ?
